superset-frontend/src/SqlLab/components/TableElement.jsx

```diff
@@ -200,8 +201,14 @@ class TableElement extends React.PureComponent {
   renderHeader() {
     const { table } = this.props;
     return (
-      <div className="clearfix">
-        <div className="pull-left">
+      <div className="clearfix header-container">
+        <Tooltip
+          id="copy-to-clipboard-tooltip"
+          placement="top"
+          style={{ cursor: 'pointer' }}
```

Comment:
Hmmm... we should probably add this to the base component. Not particularly worried about it for now, but worth considering it for another PR, since it may require some auditing of use cases.